src/System.IO.Pipelines/pkg/System.IO.Pipelines.pkgproj

```diff
@@ -7,10 +7,18 @@
     <ProjectReference Include="..\src\System.IO.Pipelines.csproj" />
     <HarvestIncludePaths Include="lib/netstandard1.3" />
   </ItemGroup>
-  <PropertyGroup>
-    <!-- Excluding the reference assets on the package so that RAR will see the run-time conflicts at build time in order to
-    generate the right binding redirects when targeting Desktop. https://github.com/dotnet/corefx/issues/32457 -->
-    <ExcludeReferenceAssets>true</ExcludeReferenceAssets>
-  </PropertyGroup>
+  <!--     This package will intentionally have a ref for netcoreapp3.0 only. In order to avoid problems when targetting dektop with RAR we will make sure there are no
```

Comment:
Do we want to move this into a targets file instead and just activate it with a property?

Reply:
Done.